magda-web-client/src/UI/DataPreviewTable.js

```diff
@@ -117,16 +135,17 @@ export default class DataPreviewTable extends Component<
             Header: item,
             accessor: item
         }));
+        const rows = this.removeEmptyRows(this.state.parsedResults.data);
```

Comment:
I'm a bit wary about looping through every row on every render - could we use `memoize-one` like in `DataPreviewMap` ?

Reply:
it actually only renders 3 times: 1st on initially request data, 2 on requesting data, 3 on receiving all data. subsequent sorting does not cause re-rendering. Changing tab do cause re-render but that is because we unmount the current tab when switching to another. Not sure how much use would `memoize-one` be in our case

Reply:
OK then 😄 🤷‍♂️